dynamodb/serverless.yml

```diff
@@ -396,6 +405,15 @@ custom:
           minimum: 40
           maximum: 40000
           usage: 0.60
+      userSwitchSearchPromptLogTable:
+        read:
+          minimum: 80
+          maximum: 40000
+          usage: 0.75
+        write:
+          minimum: 40
```

Comment:
Set minimum to 1. Will likely be a very low-write table.

Reply:
FYI, the main reasons we have minimums is: 1) to better manage a spike in throughput when we enable a feature, because the DB will need a few minutes to scale up -- so we'll set it higher for a new table with expected, immediate high traffic; and 2) to be more resilient to spiky traffic for critical tables.

Reply:
Thank you for the explanation!